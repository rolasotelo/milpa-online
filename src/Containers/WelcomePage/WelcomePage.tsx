import React from "react";
import { hot } from "react-hot-loader/root";

interface Props {}

const WelcomePage = (props: Props) => {
  return (
    <>
      <h1 className="text-4xl text-white bg-mexicanGreen hover:bg-mexicanGreen-light">
        MILPA
      </h1>
      <a href="/play">A play</a>
    </>
  );
};

export default hot(WelcomePage);
