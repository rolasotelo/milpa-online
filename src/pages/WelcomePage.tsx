import React from "react";
import {Welcome} from "../containers/Welcome";

export default function WelcomePage(){
    return (<Welcome/>)
}