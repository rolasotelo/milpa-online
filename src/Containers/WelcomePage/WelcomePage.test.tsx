import React from "react";
import renderer from "react-test-renderer";

import WelcomePage from "./WelcomePage";

describe("<WelcomePage />", () => {
  const defaultProps = {};
  const wrapper = renderer.create(<WelcomePage {...defaultProps} />);

  test("render", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
