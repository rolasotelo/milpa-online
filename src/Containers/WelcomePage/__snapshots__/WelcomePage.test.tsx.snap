// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<WelcomePage /> render 1`] = `
Array [
  <h1
    className="text-4xl text-white bg-mexicanGreen hover:bg-mexicanGreen-light"
  >
    MILPA
  </h1>,
  <a
    href="/play"
  >
    A play
  </a>,
]
`;
